### ISSUE TITLE
Google Sign In and schedule sync

### ISSUE THREAD

**Issue by n8ebel** [Droidcon-Boston/conference-app-android#127]

**Comment by n8ebel** [comment]
possibly related to #131

**Comment by fetching** [comment]
Agree related to #131 .

**Comment by kenyee** [comment]
Auth is in place once the user profile stuff is merged.
Can stuff it into the same data area under a /favorites root or something like that.

**Comment by n8ebel** [comment]
this is a lower priority for the 2019 app right now
